test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -2313,3 +2338,30 @@ class SingleDisposalsTriageControllerSpec
   }
 
 }
+
+object SingleDisposalsTriageControllerSpec extends Matchers {
+  def validateSingleDisposalTriageCheckYourAnswersPage(
+    completeSingleDisposalTriageAnswers: CompleteSingleDisposalTriageAnswers,
+    doc: Document
+  )(implicit messages: MessagesApi, lang: Lang): Unit = {
+    doc.select("#individualUserType-answer").text() shouldBe messages(
+      s"individualUserType.${completeSingleDisposalTriageAnswers.individualUserType}"
+    )
+    doc.select("#numberOfProperties-answer").text() shouldBe "One"
+    doc.select("#disposalMethod-answer").text() shouldBe messages(
+      s"disposalMethod.${completeSingleDisposalTriageAnswers.disposalMethod}"
+    )
+    if (completeSingleDisposalTriageAnswers.countryOfResidence.isUk())
+      doc.select("#wereYouAUKResident-answer").text() shouldBe "Yes"
+    else
+      doc.select("#wereYouAUKResident-answer").text() shouldBe "No"
+
+    if(completeSingleDisposalTriageAnswers.countryOfResidence.isUk())
+    completeSingleDisposalTriageAnswers.assetType match {
+      case Residential        => doc.select("#propertyType-answer").text() shouldBe "Yes"
+      case NonResidential     => doc.select("#propertyType-answer").text() shouldBe "No"
+      case IndirectDisposal   => ""
```

Comment:
you could in theory still test`doc.select("#propertyType-answer").text() should Be ""` for indirect disposals and mixed use - then at least we have a test which will fail when we actually come to do those asset types.

Reply:
done